app/scripts/components/analysis/results/index.tsx

```diff
@@ -159,7 +145,7 @@ export default function AnalysisResults() {
           </Button>
         )}
       />
-      <Fold>
+      <Fold style={{ height: '200vh' }}>
```

Comment:
I'm pretty new to styled components so asking naively: what's the rule on using inline style along styled components? On a case-by-case basis?

Reply:
Ups. This was not supposed to be here.
Answering your question: no inline styles